content/notes/config.md

```diff
@@ -23,6 +24,49 @@ links: # Links to show in footer
     link: https://github.com/jackyzha0
 ```
 
+### Code Block Titles
+
+Quartz relies on Hugo's code highlighter,
+[Chroma](https://github.com/alecthomas/chroma), which happens to be ["really,
+really fast"](https://gohugo.io/content-management/syntax-highlighting/). As
+fast as it is, it does leave much to be desired, such as code block titles. In
+Obsidian, there's a plugin which provides this feature, called [Obsidian Better
+Code Block](https://github.com/stargrey/obsidian-better-codeblock). The idea
+behind the syntax is similar to Hugo's, although unfortunately not identical
+(stay tuned for a [Quartz
+binary](https://github.com/jackyzha0/quartz/issues/81)). To add code titles
+with Quartz:
+
+1. Ensure that code block titles are enabled in Quartz's configuration:
+
+    ```yaml {title="data/config.yaml", linenos=false}
+    enableCodeBlockTitle: true
+    ```
+
+2. Add the `title` attribute to the desired code block. Note there are two ways of
+   defining code blocks in Hugo.
+
+   - [**Highlight Shortcode**](https://gohugo.io/content-management/syntax-highlighting/#example-highlight-shortcode): Example not included as Hugo will insist on attempting to render it.
```

Comment:
would just exclude this as vast majority of people will be using highlight code fences and if they use the shortcode, safe to assume they are already comfortable with Hugo to know how to add the `title` attr